vips/vip-400/bscmainnet.ts

```diff
@@ -0,0 +1,335 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+export const POOL_REGISTRY = "0x9F7b01A536aFA00EF10310A162877fd792cD0666";
+export const COMPTROLLER = "0xBE609449Eb4D76AD8545f957bBE04b596E8fC529";
+export const RESILIENT_ORACLE = "0x6592b5DE802159F3E74B2486b091D11a8256ab8A";
+export const TREASURY = "0xf322942f644a996a617bd29c16bd7d231d9f35e9";
+export const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+export const USDT = "0x55d398326f99059fF775485246999027B3197955";
+export const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+export const BTCB = "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c";
+export const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+export const WSTETH = "0x26c5e01524d2E6280A48F2c50fF6De7e52E9611C";
+export const WEETH = "0x04C0599Ae5A44757c0af6F9eC3b93da8976c150A";
+export const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+export const WSTETH_VTOKEN = "0x94180a3948296530024Ef7d60f60B85cfe0422c8";
+export const WEETH_VTOKEN = "0xc5b24f347254bD8cF8988913d1fd0F795274900F";
+export const ETH_VTOKEN = "0xeCCACF760FEA7943C5b0285BD09F601505A29c05";
+
+export const SWAP_ROUTER = "0xfb4A3c6D25B4f66C103B4CD0C0D58D24D6b51dC1";
+
+export const BOUND_VALIDATOR = "0x6E332fF0bB52475304494E4AE5063c1051c7d735";
+export const UPPER_BOUND_RATIO = parseUnits("1.01", 18);
+export const LOWER_BOUND_RATIO = parseUnits("0.99", 18);
+export const WSTETH_ONEJUMP_REDSTONE_ORACLE = "0x90dd7ae1137cC072F7740Ee0b264f2351515B98A";
+// export const WSTETH_ONEJUMP_CHAINLINK_ORACLE = "0x3C9850633e8Cb5ac5c3Da833C947E7c91EED15C4";
+export const WEETH_ONEJUMP_REDSTONE_ORACLE = "0xb661102c399630420A4B9fa0a5cF57161e5452F5";
+export const WEETH_ONEJUMP_CHAINLINK_ORACLE = "0x3b3241698692906310A65ACA199701843404E175";
+
+export const CHAINLINK_ORACLE = "0x1B2103441A0A108daD8848D8F5d790e4D402921F";
+export const WSTETH_CHAINLINK_FEED = ""; // TBD
+export const WEETH_CHAINLINK_FEED = "0xF37Be32598E9851f785acA86c2162e7C1A8466dd";
+export const CHAINLINK_STALE_PERIOD = 26 * 60 * 60; // 26 hours
+
+export const REDSTONE_ORACLE = "0x8455EFA4D7Ff63b8BFD96AdD889483Ea7d39B70a";
+export const WSTETH_REDSTONE_FEED = "0x9b2C948dbA5952A1f5Ab6fA16101c1392b8da1ab"; // TODO: use "0xF43a01A39Fd28950031a42861faD703147d5C8e7"
```

Comment:
Use `0xa76dB2Cb356ba111cCB5a7Ca369D17E1592f42Dd`, the wstETH/stETH price feed from RedStone

<img width="935" alt="Screenshot 2024-09-11 at 15 28 34" src="https://github.com/user-attachments/assets/f33c1195-86f4-49d2-8d04-7e56ecf8830c">